conf/messages.en

```diff
@@ -69,3 +70,11 @@ packagedContractPacker.detailsContent3 = will not have more than 1 million litre
 packagedContractPacker.checkYourAnswersLabel = packagedContractPacker
 packagedContractPacker.error.required = Select yes if you are reporting liable drink you have packaged as a third party or contract packer at UK sites you operate
 packagedContractPacker.change.hidden = packagedContractPacker
+
+ownBrands.title = Are you reporting your own brands of liable drinks packaged at UK sites you operate?
+ownBrands.heading = Are you reporting your own brands of liable drinks packaged at UK sites you operate?
+ownBrands.subHeading = This includes brands you own or have the rights to manufacture.
+ownBrands.checkYourAnswersLabel = ownBrands
+ownBrands.error.required = Select yes if ownBrands
```

Comment:
I think you need to edit these messages . They should match with existing messages